test/e2e/testdata/retry-workflow-with-continueon.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: "dag-to-retry"
```

Comment:
I would rename this to align with the filename

Reply:
renamed.